diff -urN krfb-4.11.0/krfb/CMakeLists.txt krfb-4.11.0-patched/krfb/CMakeLists.txt
--- krfb-4.11.0/krfb/CMakeLists.txt	2013-08-19 22:28:14.357525758 +1100
+++ krfb-4.11.0-patched/krfb/CMakeLists.txt	2013-08-19 22:28:45.291038423 +1100
@@ -24,7 +24,7 @@
 
 set_target_properties (krfbprivate PROPERTIES
                        VERSION ${GENERIC_LIB_VERSION}
-                       SOVERSION ${GENERIC_LIB_VERSION}
+                       SOVERSION ${GENERIC_LIB_SOVERSION}
 )
 
 install (TARGETS krfbprivate
